apps/meteor/client/views/room/contextualBar/TeamsVoipConfigModal.tsx

```diff
@@ -0,0 +1,100 @@
+import { Modal, Button, Box, Callout, Margins } from '@rocket.chat/fuselage';
+import type { ReactElement } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { useExternalLink } from '../../../hooks/useExternalLink';
+import { GET_ADDONS_LINK } from '../../admin/subscription/utils/links';
+
+type TeamsVoipConfigModalProps = {
+	onClose: () => void;
+	onConfirm?: () => void;
+	isAdmin: boolean;
+	hasModule: boolean;
+};
+
+const TeamsVoipConfigModal = ({ onClose, onConfirm, isAdmin, hasModule }: TeamsVoipConfigModalProps): ReactElement => {
+	const { t } = useTranslation();
+	const openExternalLink = useExternalLink();
+
+	const getCalloutWarning = () => {
+		if (isAdmin && !hasModule) {
+			return t('Contact_sales_start_using_VoIP');
+		}
+
+		if (!isAdmin && !hasModule) {
+			return t('Contact_your_workspace_admin_to_start_using_VoIP');
+		}
+
+		return t('VoIP_available_setup_freeswitch_server_details');
+	};
+
+	return (
+		<Modal>
+			<Modal.Header>
```

Comment:
I'd recommend using the named exports instead (Same for all other `Modal.*`). Even though we haven't deprecated this way of using it for modals, we already did for the `Field` component. Would be a good way to future proof it.
```suggestion
			<ModalHeader>
```